src/api/index.ts

```diff
@@ -1,23 +1,18 @@
 import axios, { AxiosError, AxiosInstance } from 'axios';
 import { Network } from '../config/enums/network.enum';
 import {
+  DateIsoFormat,
   GasPrices,
   MerkleProof,
   PriceSummary,
   TokenConfiguration,
 } from './types';
-import {
-  Account,
-  APIOptions,
-  LeaderboardSummary,
-  ProtocolMetrics,
-  ProtocolSummary,
-  Vault,
-} from './interfaces';
+import * as i from './interfaces';
```

Comment:
is there a reason to do it this way vs. modular imports? not sure if i like this lower case i but also you don't see things like this in java

Reply:
it`s not the rule, just i`m doing * all the time i have imports > 5. To keep file readability. We can keep it your way for consistency